examples/platforms/cc2538/radio.c

```diff
@@ -704,6 +729,14 @@ static void readFrame(void)
     {
         sReceiveFrame.mLength            = length;
         sReceiveFrame.mInfo.mRxInfo.mLqi = crcCorr & CC2538_LQI_BIT_MASK;
+
+        if (length > IEEE802154_ACK_LENGTH)
+        {
+            // Set ACK FP flag for the received frame according to whether SRC_MATCH_FOUND was triggered just before
+            // if SRC MATCH is not enabled, SRC_MATCH_FOUND is not triggered and all ACK FP is always set
+            sReceiveFrame.mInfo.mRxInfo.mAckedWithFramePending =
+                cc2538SrcMatchEnabled() ? cc2538GetSrcMatchFoundIntFlag() : true;
```

Comment:
should it be set to false if `cc2538SrcMatchEnabled()` is not enabled?

Reply:
recall previous comment as it is PENDING_OR when disabled

Reply:
When source match table is not enabled, ack FP is always set according to datasheet